golang/lib/services/service.go

```diff
@@ -21,3 +21,24 @@ package services
 The identifier used for services with the network.
 */
 type ServiceID string
+
+
+type SharedDirectory struct {
+	//The service directory absolute path in the testsuite container
+	absFilepathOnThisContainer string
+	//The service directory absolute path in the service container
+	absFilepathOnServiceContainer string
+}
+
+func NewSharedDirectory(absFilepathOnThisContainer string, absFilepathOnServiceContainer string) *SharedDirectory {
+	return &SharedDirectory{absFilepathOnThisContainer: absFilepathOnThisContainer, absFilepathOnServiceContainer: absFilepathOnServiceContainer}
+}
+
+func (s SharedDirectory) GetAbsFilepathOnThisContainer() string {
```

Comment:
Same thing here and below - would recommend documenting this like anything else in this API